include/aspect/particle/distribution.h

```diff
@@ -93,27 +93,38 @@ namespace aspect
          * @param particle_range The particle_iterator_range to operate on.
          * @param particle_ranges_to_sum_over The ranges of the particles in neighboring cells.
```

Comment:
Update this parameter documentation. This parameter no longer just contains the particles in neighboring cells, it also contains the particles in the current cell.